### PR TITLE
Adding hometown file for Mogle1

### PR DESCRIPTION
**Pull request by Mogle1**
@jrdisc2 - What kind of restaurants do you like so that I can add them to the hometown file?

### CODE REVIEW

**Review by jrdisc2 — COMMENTED** [review]
This is a review you requested from me

File: mogle1-milwaukee.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Milwaukee, WI
+
+### Restaurants
+
```
Comment (jrdisc2) [inline mogle1-milwaukee.md:4]: What about Culvers :hamburger: , they're from Wisconsin right? :cheese: Cool!!! :sunglasses:

Comment (Mogle1) [reply mogle1-milwaukee.md:4]: @jrdisc2 added some restaurants, let me know if this is good to :ship:

Comment (jrdisc2) [reply mogle1-milwaukee.md:4]: You bet! :sunglasses:

**Review by jrdisc2 — APPROVED** [review]
Approved changes!

**Pull request merged**